processing/src/main/java/io/druid/segment/column/DictionaryEncodedColumn.java

```diff
@@ -34,4 +37,6 @@
   public ActualType lookupName(int id);
   public int lookupId(ActualType name);
   public int getCardinality();
+
+  HistoricalDimensionSelector makeDimensionSelector(OffsetHolder offsetHolder, ExtractionFn extractionFn);
```

Comment:
Is there a way to architect this so there isn't a historical-specific method in `DictionaryEncodedColumn`?